modules/apps/portal-search/portal-search-web-api/src/main/java/com/liferay/portal/search/web/portlet/shared/search/PortletSharedSearchSettings.java

```diff
@@ -30,14 +30,10 @@
 @ProviderType
 public interface PortletSharedSearchSettings extends SearchSettings {
 
-	public Optional<String> getParameter71(String name);
-
-	public Optional<String> getParameterOptional(String name);
+	public String getParameter(String name);
```

Comment:
@julschong Why does this interface still using `Optional` for `getPortletPreferences*`?

Reply:
@dantewang getPortletPreference portion will be on part 3, there is a lot to change for that so will be the final part for this module